packages/contracts/src/EmailAccountRecovery.sol

```diff
@@ -124,45 +110,49 @@ abstract contract EmailAccountRecovery {
     /// @param recoveredAccount The address of the account to be recovered.
     /// @param accountSalt A bytes32 salt value defined as a hash of the guardian's email address and an account code. This is assumed to be unique to a pair of the guardian's email address and the wallet address to be recovered.
     /// @return address The computed address.
-    function computeEmailAuthAddress(
+    function computeProxyAddress(
```

Comment:
If you replace `computeEmailAuthAddress ` with `computeProxyAddress `, could you name this function `computeEmailAuthAddress`?

Reply:
@SoraSuegami Does that mean we leave the function name as `computeEmailAuthAddress ` ?